### DISCUSSION TITLE
Implement Nested Persisting Layout Between Page Changes in each small app separately

### DISCUSSION THREAD

**Discussion by winner106** [blitz-js/blitz#3079] [Q&A] [answered]
Thank you very much for sharing this wonderful project. I have been learning the `blitz` documentation for a while, so I recently plan to try to use `blitz` to do a comprehensive project.

How to Implement Nested Persisting Layout Between Page Changes?

Hope to get your suggestions, thank you very much!

**Comment by abuuzayr** [comment, marked as answer]
hi @winner106 can you clarify what you’re asking? It doesn’t seem so clear. Perhaps you can share your use case?
